apps/election-manager/src/components/Table.tsx

```diff
@@ -30,4 +30,10 @@ export const TD = styled.td<TableDataProps>`
   white-space: ${({ nowrap }) => (nowrap ? 'nowrap' : undefined)};
 `
 
+export const TH = styled.th<TableDataProps>`
+  width: ${({ narrow = false }) => (narrow ? '1%' : undefined)};
+  text-align: ${({ textAlign }) => textAlign};
+  white-space: ${({ nowrap }) => (nowrap ? 'nowrap' : undefined)};
+`
+
```

Comment:
Rather than duplicating the TD component functionality, we've been implementing like this elsewhere:

```
<TD as="th" narrow nowrap>
```

If you'd rather have a separate `TH` component we should pull out the shared styles into a shared variable which each uses.

Here's a summary of techniques for tag agnostic styled components: https://stackoverflow.com/questions/42204232/how-to-achieve-tag-agnostic-styled-components

Reply:
Ahhh I knew we were doing this with Th's somewhere but couldn't find it! Thanks!